packages/next-config/withWebSecurityHeaders.ts

```diff
@@ -39,7 +39,7 @@ export function withWebSecurityHeaders(config: NextConfig): NextConfig {
           },
           {
             key: 'Cross-Origin-Opener-Policy',
-            value: 'same-origin',
+            value: 'same-origin-allow-popups',
```

Comment:
let's modify in the web app and keep here untouched. Ideally all apps should use the 'same-origin' if there's no special requirement